sys/include/net/coap.h

```diff
@@ -27,9 +27,12 @@ extern "C" {
 #endif
 
 /**
- * @brief   Default CoAP port
+ * @name    Default CoAP ports
+ * @{
  */
-#define COAP_PORT               (5683)
+#define COAP_PORT               (5683)  /**< unencrypted CoAP port */
+#define COAPS_PORT              (5684)  /**< CoAP over DTLS port */
```

Comment:
This is already defined just a few lines below

Reply:
Oops, I think git somehow managed to merge this in without asking me ^^"